python/requirements.txt

```diff
@@ -1,6 +1,6 @@
 msgpack==1.0.0
 loguru==0.5.3
 requests==2.24.0
-requests-http-signature==0.1.0
+requests-http-signature==0.2.0
```

Comment:
A new version of this was released a while back: https://github.com/pyauth/requests-http-signature/releases (at least on PyPi).